apps/finance/artifact.json

```diff
@@ -984,11 +984,11 @@
 			"notice": "Execute pending payment #`_paymentId`"
 		},
 		{
-			"sig": "setPaymentDisabled(uint256,bool)",
+			"sig": "setPaymentStatus(uint256,bool)",
 			"roles": [
 				"DISABLE_PAYMENTS_ROLE"
 			],
-			"notice": "`_disabled ? 'Disable' : 'Enable'` payment `_paymentId`"
+			"notice": "`_active ? 'Active' : 'Inactive'` payment `_paymentId`"
```

Comment:
@izqui I'm thinking we should just remove this file from the repo for now, and add it back when we're getting closer to deployment. We'll have to regen it anyway at that point.

Reply:
Hmm, I don't know if there's a difference between the two. I kind of feel like enable/disable is better.

Reply:
I'd say active/inactive makes more sense if we choose `status` for the name of the function, because saying a payments status is `disable` is weird, but `inactive` sounds totally fine.

Reply:
Agreed, thought about it some more and it makes more sense if you can re-enable recurring payments ("active recurring payment" sounds a lot better than "enabled recurring payment").